In [2]:
from curves.persistence_length import BigWindowPersistenceLength, SmallWindowPersistenceLength

In [34]:
rootfolder = '/home/ytcdata/curves_data'
host = 'tgtg_21mer'

In [22]:
big_agent = BigWindowPersistenceLength(rootfolder, host)

In [ ]:
#big_agent.make_lp_big_wn_csv()
big_agent.read_lp_big_wn_csv()
big_agent.df_lp

In [35]:
if host in ['pnas_dna', 'pnas_rna']:
    n_frames_per_window = 2000
else:
    n_frames_per_window = 5000
#n_frames_per_window = 4

small_agent = SmallWindowPersistenceLength(rootfolder, host, n_frames_per_window)

Read /home/ytcdata/curves_data/tgtg_21mer/bend_shape/fourier_amplitude.hdf5


In [ ]:
#small_agent.make_lp_small_wn_csv()
small_agent.read_lp_small_wn_csv()
small_agent.df_lp

In [38]:
mode_id = 2
small_agent.print_avg_std(mode_id)

Mode-ID: 2
Lp =49.3±5.6


### Table

|          | mean | std |
|----------|------|-----|
| pnas_dna | 60.8 | 6.3 |
| pnas_rna | 48.4 | 1.8 |
| polyA    | 71.8 | 3.8 |
| polyG    | 49.1 | 2.6 |
| TpA      | 53.3 | 5.8 |
| CpG      | 66.5 | 2.7 |
| TpC      | 52.2 | 2.5 |
| GpT      | 49.3 | 5.6 |